app/models/labware_creators/plate_split_to_tube_racks.rb

```diff
@@ -0,0 +1,453 @@
+# frozen_string_literal: true
+
+require_dependency 'form'
+require_dependency 'labware_creators/base'
+
+module LabwareCreators
+  # Handles the creation of up to 2 child racks of tubes from a single parent 96-well plate.
+  # Intended for use in a PBMC cell extraction pipeline.
+  #
+  # There will typically be one rack of tubes for 'contingency' and one for 'sequencing' (this rack is
+  # optional).
+  #
+  # The parent plate contains multiple copies of material prepared from the same sample. One well instance
+  # of each sample will go into a tube in the 'sequencing' rack (if present), and any remaining copies
+  # will go into tubes in the 'contingency' rack.
+  #
+  # If after the initial preparation the users feel they need more contingency tubes, then they will go
+  # back to prepare more material for that sample from an earlier step in the pipeline and create just
+  # 'contingency' tubes at this step.
+  #
+  # Inputs:
+  # 1) The parent plate - contains a number of wells containing material prepared from the same sample.
+  #    The first of these parent wells will be transferred into a tube in the 'sequencing' rack if it is
+  #    present, and any remaining parent wells for the same sample will be transferred into tubes in the
+  #    'contingency' rack.
```

Comment:
It's not quite clear here and in the paragraph starting on line 13 that there will be multiple groups like this. 
`contains groups of wells containing material prepared from the same sample.` - would that be clearer maybe?

Reply:
changed